setup.py

```diff
@@ -74,12 +74,12 @@ def get_extras_require() -> Dict[str, List[str]]:
             "scikit-image",
             "scikit-learn",
             "thop",
-            "torch==1.4.0" if sys.platform == "darwin" else "torch==1.4.0+cpu",
-            "torchvision==0.5.0" if sys.platform == "darwin" else "torchvision==0.5.0+cpu",
+            "torch==1.5.1" if sys.platform == "darwin" else "torch==1.5.1+cpu",
+            "torchvision==0.6.1" if sys.platform == "darwin" else "torchvision==0.6.1+cpu",
             "xgboost",
         ]
         + (
-            ["allennlp<1", "fastai<2", "pytorch_lightning>=0.7.1"]
+            ["allennlp==1.0.0", "fastai<2", "pytorch-lightning>=0.7.1"]
```

Comment:
What's the difference between "pytorch_lightning" and "pytorch-lightning"?

Reply:
Thank you for pointing out, I unintentionally introduced this change. I removed it in 7143a3e.

BTW, I found `pytorch-lightning` and `pytorch_lightning` both appear in `setup.py`.
(I tested `pip install` them and confirmed it downloaded `pytorch_lightning` in both cases)
https://github.com/optuna/optuna/pull/1392/commits/f53cbe225032bed25d61a06db0f5250243d3d348#diff-2eeaed663bd0d25b7e608891384b7298R82-R88

According to PyPI, `pytorch-lightning` seems to be right.
https://pypi.org/project/pytorch-lightning/

Reply:
I side for `pytorch-lightning`. Thanks for checking PyPI.